NachoSpigot-Server/src/main/java/net/minecraft/server/ItemBucket.java

```diff
@@ -96,18 +94,6 @@ public ItemStack a(ItemStack itemstack, World world, EntityHuman entityhuman) {
 
                     if (this.a(world, blockposition1) && !entityhuman.abilities.canInstantlyBuild) {
                         entityhuman.b(StatisticList.USE_ITEM_COUNT[Item.getId(this)]);
-                        // PaperSpigot start - Stackable Buckets
-                        if ((this == Items.LAVA_BUCKET && PaperSpigotConfig.stackableLavaBuckets) ||
```

Comment:
Why remove this?

Reply:
It doesn't work since it's functionality was removed to stop errors in Java 12+